### PR TITLE
Add plugin requisites to the readme file

### PR DESCRIPTION
**Pull request by DAreRodz**
Adds a **Requisites** section in the readme, containing the same info that is in the [documentation](https://docs.frontity.org/frontity-plugins/rest-api-head-tags#requisites).

### CODE REVIEW

File: plugins/frontity-headtags/readme.txt [hunk 1]
```diff
@@ -16,6 +16,12 @@ This plugin adds all the tags in the head section of a website to WordPress REST
 
 It is perfect if you are using WordPress for a headless set-up and would like to add the **meta tags** generated by your **WordPress SEO plugin** (like Yoast SEO or All-in-One SEO Pack) to the WordPress REST API output.
 
+#### Requisites
```
Comment (luisherranz) [inline plugins/frontity-headtags/readme.txt:19]: @mburridge is requisites the correct word here? I'm more used to "Requirements", but I'd love to know your opinion.

**Comment by SantosGuillamot** [comment]
Do we need a changeset after changing the `readme`?

**Comment by luisherranz** [comment]
No. Changeset is only for changes afecting code.

**Comment by DAreRodz** [comment]
I'm going to wait for @mburridge to give his opinion on [this](https://github.com/frontity/wp-plugins/pull/36#discussion_r429557658) before merging and deploying.

File: plugins/frontity-headtags/readme.txt (hunk 1 above)
Comment (mburridge) [reply plugins/frontity-headtags/readme.txt:19]: I would use either 'Requirements' or 'Prerequisites'.

**Comment by mburridge** [comment]
@DAreRodz I've replied [here](https://github.com/frontity/wp-plugins/pull/36#discussion_r430277802)

File: plugins/frontity-headtags/readme.txt (hunk 1 above)
Comment (DAreRodz) [reply plugins/frontity-headtags/readme.txt]: Changed.

**Pull request merged**